storage/range.go

```diff
@@ -235,13 +235,35 @@ func (r *Range) String() string {
 	return fmt.Sprintf("range=%d (%s-%s)", r.Desc().RaftID, r.Desc().StartKey, r.Desc().EndKey)
 }
 
-// start begins by gossiping the cluster ID (only happens if the range
-// is the first in the key space). This is necessary to get the cluster
-// started from scratch.
+// start begins by gossiping the configs. It also gossips the sentinel if we're
+// the first range (the store will do this regularly, but this is a good time
+// to get started quickly).
 func (r *Range) start() {
-	if r.IsFirstRange() {
-		r.startGossip(r.rm.Stopper())
+	// Being the first range comes with extra responsibility for the gossip
```

Comment:
yes, intent of the long comment below is to land this and then embark on a fun journey about this. We'll need to figure out how exactly we wanna do this.